web/client/translations/data.it-IT.json

```
@@ -3714,6 +3718,70 @@
         },
         "sidebarMenu": {
           "showMoreItems": "Mostra più elementi"
+        },
+        "longitudinalProfile": {
+          "open": "Apri profilo longitudinale",
+          "close": "Chiudi profilo longitudinale",
+          "title": "Profilo longitudinale",
+          "draw": "Disegna profilo",
+          "import": "Carica file",
+          "select": "Seleziona il livello",
```

Something like "Seleziona da livello" "Linea da livello" or  "Profilo da livello" sounds better in italian, for the menu entry.
```suggestion
          "select": "Profilo da livello",
```